Gemfile.lock

```diff
@@ -177,7 +179,7 @@ GEM
       railties (>= 3.0.0)
     faker (1.7.3)
       i18n (~> 0.5)
-    ffi (1.12.2)
+    ffi (1.13.0)
```

Comment:
### [ffi](https://github.com/ffi/ffi/wiki)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.12.2 → 1.13.0

[_[change-log](https://github.com/ffi/ffi/blob/master/CHANGELOG.md), [source-code](https://github.com/ffi/ffi/)_]





<details>
<summary>Commits</summary>

A change of **114** commits. See the full changes on [the compare page](https://github.com/ffi/ffi/compare/1.12.2...1.13.0).

These are the first 10 commits:
- (2d9f010) [Replace legacy caddr_t by void* or char*](https://github.com/ffi/ffi/commit/2d9f010228184d9a3adcd4eb2f6f14c1a8ad52c4)
- (db860de) [Avoid warning about argument type of rb_define_singleton_met…](https://github.com/ffi/ffi/commit/db860deca67e474a1c2ca0c89304aa17ea548685)
- (333da91) [Avoid warning about incompatible parameter type](https://github.com/ffi/ffi/commit/333da91e89b571f7a38c7743b7f5db661e6ad342)
- (143c72f) [Avoid warning about parameter type](https://github.com/ffi/ffi/commit/143c72f020f2c59b4ea7869818d753769ed78d42)
- (0fc7869) [replace ffi_prep_closure with ffi_prep_closure_loc](https://github.com/ffi/ffi/commit/0fc78699ff2d4dfe19391c43efd2d3553689feb8)
- (83c729f) [Load JRuby ext under JRuby](https://github.com/ffi/ffi/commit/83c729ff6a4ff82bc0c0790118e3b959e617146c)
- (40383fe) [Remove MACOSX_DEPLOYMENT_TARGET in FFI Makefile](https://github.com/ffi/ffi/commit/40383fec7eb78fa66f90d32cb5dcf72a6f45c94d)
- (13e8ab6) [Always call make for FFI specs](https://github.com/ffi/ffi/commit/13e8ab6dea4d9777ef3998f44c62c0207ccc93e0)
- (0b4e779) [Update to latest libffi](https://github.com/ffi/ffi/commit/0b4e779cb02bce566ad076362b019c462954e1fa)
- (b7f3bad) [Merge pull request #754 from eregon/make-fixes](https://github.com/ffi/ffi/commit/b7f3bad80953be567c79bfddc5a18988e6464676)


</details>